File: app/src/main/java/net/nashihara/naroureader/entities/Novel4Realm.java
package net.nashihara.naroureader.entities;

import io.realm.RealmObject;
import io.realm.annotations.PrimaryKey;

public class Novel4Realm extends RealmObject {
    @PrimaryKey
    private String ncode;
    private String title;
    private String writer;
    private int bookmark;
    private boolean fav;

    public String getNcode() {
        return ncode;
    }

    public void setNcode(String ncode) {
        this.ncode = ncode;
    }

    public String getTitle() {
        return title;
    }

    public void setTitle(String title) {
        this.title = title;
    }

    public String getWriter() {
        return writer;
    }

    public void setWriter(String writer) {
        this.writer = writer;
    }

    public int getBookmark() {
        return bookmark;
    }

    public void setBookmark(int bookmark) {
        this.bookmark = bookmark;
    }

    public boolean isFav() {
        return fav;
    }

    public void setFav(boolean fav) {
        this.fav = fav;
    }
}

File: app/src/main/java/net/nashihara/naroureader/entities/Novel4Realm.kt
package net.nashihara.naroureader.entities

import io.realm.RealmObject
import io.realm.annotations.PrimaryKey
import io.realm.annotations.RealmClass

@RealmClass
open class Novel4Realm(
        _ncode: String? = null,
        var title: String? = null,
        var writer: String? = null,
        var story: String? = null,
        var bookmark: Int = 0,
        var fav: Boolean = false,
        var totalPage: Int = 0,
        var isDownload: Boolean = false) : RealmObject() {

    @PrimaryKey
    var ncode: String? = _ncode

    override fun toString(): String {
        return "Novel4Realm{" +
                "ncode='" + ncode + '\'' +
                ", title='" + title + '\'' +
                ", writer='" + writer + '\'' +
                ", story='" + story + '\'' +
                ", bookmark=" + bookmark +
                ", fav=" + fav +
                ", totalPage=" + totalPage +
                ", isDownload=" + isDownload +
                '}'
    }
}
